### PR TITLE
Small fixes, typos and other changes

### PR DESCRIPTION
**Pull request by lexnederbragt**
- Explains terms not every reader may know
- Fixes typos
- Adds a few `fixme`s

### CODE REVIEW

File: good-enough-practices-for-scientific-computing.tex [hunk 1]
```diff
@@ -57,12 +57,11 @@
...
-Two years ago a group of researchers involved in \withurl{Software
-  Carpentry}{http://software-carpentry.org/} and \withurl{Data
-  Carpentry}{http://datacarpentry.org/} wrote a paper called
-``\withurl{Best Practices for Scientific
-  Computing}{http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1001745}''
-\cite{wilson2014}. It was well received, but many novices found its
+Two years ago a group of researchers involved in \withurl{Software 
+Carpentry}{http://software-carpentry.org/} and \withurl{Data 
+Carpentry}{http://datacarpentry.org/} wrote a paper called
+``Best Practices for Scientific Computing''\cite{wilson2014}.
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex]: space before \cite

File: good-enough-practices-for-scientific-computing.tex [hunk 2]
```diff
@@ -132,7 +129,7 @@ \section{Data Management}\label{sec:data}
   format that is inefficient to store or process, transform it for
   storage with a lossless, well-documented procedure.  Equally, don't
   duplicate contents of stable, long-lived repositories (i.e., don't
-  clone GenBank).
+  download the entire DNA sequence database GenBank).
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex:132]: I prefer the original, but will defer to others.

File: good-enough-practices-for-scientific-computing.tex [hunk 3]
```diff
@@ -208,7 +205,8 @@ \section{Data Management}\label{sec:data}
   \begin{framed}
     \noindent \textbf{OpenRefine}
 
-    \withurl{OpenRefine}{http://openrefine.org/} is an excellent tool
+    \withurl{OpenRefine}{http://openrefine.org/} \fixme{LN}{url from frame does not appear in footnote}
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex]: I hate LaTeX :-(

File: good-enough-practices-for-scientific-computing.tex [hunk 4]
```diff
@@ -246,10 +244,12 @@ \section{Data Management}\label{sec:data}
 \item
   \recommend{Include complementary data in files that meet the same
     standards}.  Raw data frequently does not fully explain
-  itself. For example, if each well of a microtitre plate is used to
+  itself. For example, if each of wells in the 8 rows by 12 columns
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex:247]: More detail than an made-up example requires?

File: good-enough-practices-for-scientific-computing.tex [hunk 5]
```diff
@@ -371,7 +371,7 @@ \section{Software}\label{sec:software}
   a dozen parameters.  The key motivation here is to fit the program
   into the most limited memory of all: ours. Human short-term memory
   is famously incapable of holding more than about seven (plus or
-  minus two) items at once. If we are to understand what our software
+  minus two)\fixme{LN}{add reference?} items at once. If we are to understand what our software
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex]: https://en.wikipedia.org/wiki/The_Magical_Number_Seven,_Plus_or_Minus_Two

File: good-enough-practices-for-scientific-computing.tex [hunk 6]
```diff
@@ -401,7 +401,7 @@ \section{Software}\label{sec:software}
     Almost all modern text editors provide \emph{tab completion}, so
     that typing the first part of a variable name and then pressing
     the tab key inserts the completed name of the variable.  Employing
-    this means that meaningful variable names are no harder to type
+    this means that meaningful longer variable names are no harder to type
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex:404]: "meaningful variable names are no harder to type than terse abbreviations" maybe?

File: good-enough-practices-for-scientific-computing.tex [hunk 7]
```diff
@@ -427,7 +427,7 @@ \section{Software}\label{sec:software}
     One special case of this recommendation is to use a logging
     library such as Java's \texttt{log4j} for debugging messages. When
     this is done, messages are left in the code, but their activity is
-    controlled by an external configuration file.
+    controlled by an external configuration file.\fixme{LN}{Can this be explained better? I didn't really understand how this works}
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex]: Please change the initials from LN to GW and I'll tackle it.

File: good-enough-practices-for-scientific-computing.tex [hunk 8]
```diff
@@ -443,7 +443,7 @@ \section{Software}\label{sec:software}
   \recommend{Submit code to a reputable DOI-issuing repository} upon
   submission of paper, just as you do with data. Your software is as
   much a product of your research as your papers, and should be as
-  easy for people to credit.
+  easy for people to credit.\fixme{LN}{add examples, e.g. zenodo and figshare?}
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex]: Yes please.

File: good-enough-practices-for-scientific-computing.tex [hunk 9]
```diff
@@ -580,7 +580,7 @@ \section{Collaboration}\label{sec:collaboration}
   doi={10.1371/journal.pone.0065848}
 }
 \end{verbatim}
-\caption{Example CITATION File}
+\caption{Example CITATION File with the information in two forms, as a regular citation and in BiBTeX format\fixme{LN}{explain BiBTex?}}
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex:585]: How about just "Example CITATION File With Information in Two Forms" ?

File: good-enough-practices-for-scientific-computing.tex [hunk 10]
```diff
@@ -1161,7 +1162,7 @@ \subsection*{Why Two?}
 have to explain the difference between plain text and word
 processing. And text editors. And markdown/LaTeX compilers. And
 BiBTeX. And Git. And GitHub. Etc. Meanwhile he/she is getting paged
-from the OR{\ldots}
+from the Operating Room{\ldots}
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex]: Have to keep the original (it's a quotation).

File: good-enough-practices-for-scientific-computing.tex (hunk 2 above)
Comment (lexnederbragt) [inline good-enough-practices-for-scientific-computing.tex:132]: Many of our intended audience will not know what GenBank is...

File: good-enough-practices-for-scientific-computing.tex (hunk 4 above)
Comment (lexnederbragt) [inline good-enough-practices-for-scientific-computing.tex:247]: Again, how many readers will know what a microtiter plate is?

File: good-enough-practices-for-scientific-computing.tex (hunk 5 above)
Comment (lexnederbragt) [inline good-enough-practices-for-scientific-computing.tex]: Ref [1] there is http://psychclassics.yorku.ca/Miller/ (or http://psycnet.apa.org/journals/rev/63/2/81/), perhaps use that one instead? Added and assigned fixme to Greg (OK?)

File: good-enough-practices-for-scientific-computing.tex (hunk 9 above)
Comment (lexnederbragt) [inline good-enough-practices-for-scientific-computing.tex:585]: Thats was my first thought, but then I realised many of our readers will not recognise the second format...

File: good-enough-practices-for-scientific-computing.tex (hunk 10 above)
Comment (lexnederbragt) [inline good-enough-practices-for-scientific-computing.tex]: OK, but there will be readers that don't know what OR means...

File: good-enough-practices-for-scientific-computing.tex (hunk 3 above)
Comment (lexnederbragt) [inline good-enough-practices-for-scientific-computing.tex]: Will file an issue

File: good-enough-practices-for-scientific-computing.tex (hunk 2 above)
Comment (beroe) [inline good-enough-practices-for-scientific-computing.tex:132]: I get the point, but we actually download the entire database from GenBank to our server on a regular basis, to run local bulk searches.

Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex:132]: Fixed in ec3ae5c.

**Pull request merged**